qualtricssurvey/public/edit.html

```diff
@@ -1 +1,3 @@
\ No newline at end of file
+<div class="wrapper-comp-settings is-active editor-with-buttons qualtricssurvey_edit" id="settings-tab"><ul class="list-input settings-list"><li class="field comp-setting-entry is-set"><div class="wrapper-comp-setting"><label class="label setting-label" for="xblock_qualtricssurvey_display_name">Display Name</label><input class="input setting-input" id="xblock_qualtricssurvey_display_name" value="{self.display_name}" tabindex="1"></div><span class="tip setting-help">This name appears in the horizontal navigation at the top of the page.</span></li><li class="field comp-setting-entry is-set"><div class="wrapper-comp-setting"><label class="label setting-label" for="xblock_qualtricssurvey_survey_id">Survey ID</label><input class="input setting-input" id="xblock_qualtricssurvey_survey_id" value="{self.survey_id}" tabindex="2"></div><span class="tip setting-help">This is the ID that Qualtrics uses for the survey, usually an 8 digit number.</span></li><li class="field comp-setting-entry is-set"><div class="wrapper-comp-setting"><label class="label setting-label" for="xblock_qualtricssurvey_your_university">Your university</label><input class="input setting-input" id="xblock_qualtricssurvey_your_university" value="{self.your_university}" tabindex="3"></div><span class="tip setting-help">This is the name of your university.</span></li><li class="field comp-setting-entry is-set"><div class="wrapper-comp-setting"><label class="label setting-label" for="xblock_qualtricssurvey_link_text">Link text</label><input class="input setting-input" id="xblock_qualtricssurvey_link_text" value="{self.link_text}" tabindex="4"></div><span class="tip setting-help">This is the text that will link to your survey.</span></li><li class="field comp-setting-entry is-set"><div class="wrapper-comp-setting"><label class="label setting-label" for="xblock_qualtricssurvey_param_name">Parameter Name for UserID</label><input class="input setting-input" id="xblock_qualtricssurvey_param_name" value="{self.param_name}" tabindex="5"></div><span class="tip setting-help">This is the name for the User ID parameter in the url. If blank, User ID is ommitted from the url.</span></li><li class="field comp-setting-entry is-set"><a href="#" class="xblock_qualtricssurvey_display-source">View Source</a><div class="content-source-div"><textarea disabled contenteditable="false" class="content-source-textarea">
```

Comment:
I think I must be missing something, but why are this file and qualtricssurvey/public/view.html expressed with so few newlines?